workinggroups/storage.md

```diff
@@ -0,0 +1,32 @@
+# CNCF Storage WG Proposal
+
+## TOC Sponsor
+
+Ben Hindman
+
+## Objective
+
+Explore cloud native storage technology and concepts.
```

Comment:
Same comment on "explore".

But we've also had discussions about what "storage" should encompass. I think the scope needs to be clarified and narrowed. We also need clarification on what the group will produce.

cc @benh @monadic @skamille